pkg/openstack/routes.go

```diff
@@ -437,11 +450,20 @@ func (r *Routes) DeleteRoute(ctx context.Context, clusterName string, route *clo
 		defer onFailure.call(unwind)
 	}
 
+	if !r.allowedAddressPairs {
+		klog.V(4).Infof("Route deleted (skipping the allowed_address_pairs update): %v", route)
+		onFailure.disarm()
+		return nil
+	}
+
 	// get the port of addr on target node.
 	port, err := getPortByIP(r.network, addr, r.networkIDs)
 	if err != nil {
 		return err
 	}
+	if !port.PortSecurityEnabled {
+		klog.V(4).Infof("Skipping allowed_address_pair for port: %v", port.ID)
```

Comment:
not sure why the `port.PortSecurityEnabled ` is involved here and seems only a log for it 
nothing impact logic, is this expected code logic?

Reply:
As I can see from [Neutron source code](https://github.com/openstack/neutron/blob/58dcd30dbba67464f6fd7880ce7aee543156af65/neutron/plugins/ml2/plugin.py#L1510), it will reject changes to `allowed_address_pairs` if port security is disabled for port

Reply:
ok, thanks for the link

Reply:
if the port security is disabled, why don't you just return?

Reply:
@kayrus 
Should we log a warning message before returning, if the port has `port_security_enabled=true` while `allowed_address_pairs` extension is disabled? If I'm not mistaken, Neutron will block traffic from pods, as it is not aware of such ips.

Reply:
I propose to change the `Infof` with a warning, then just return.